10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
 ### 2 Samples, 2 Seq-Libraries, 1 Seq-Run/Flowcell, 2 FBMs
 [![Schematic of 2-sample, 1-seq-run workflow](https://support.10xgenomics.com/img/mkfastq-1.png "")](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq#example_workflows)
 
-> In this example, we have two 10x libraries (each processed through a separate Chromium chip channel) that are multiplexed on a single flowcell. Note that after running cellranger mkfastq, we run a separate instance of the pipeline on each library"
+> In this example, we have two 10x libraries (each processed through a separate Chromium chip channel) that are multiplexed on a single flowcell. Note that after running `cellranger mkfastq`, we run a separate instance of the pipeline on each library
 
 ### 1 Sample, 1 Seq-Library, 2 Seq-Run/Flowcells, 1 FBM
 [![Schematic of 1-sample, 2-seq-run workflow](https://support.10xgenomics.com/img/mkfastq-2.png "")](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq#example_workflows)
-> In this example, we have one 10x library sequenced on two flowcells. Note that after running cellranger mkfastq, we run a single instance of the pipeline on all the FASTQ files generated"
+> In this example, we have one 10x library sequenced on two flowcells. Note that after running cellranger mkfastq, we run a single instance of the pipeline on all the FASTQ files generated
 
-The common scenario of sequencing the same 'sequencing library' more than once (in the above example) is the reason why the 10x technicians are moving towards a FASTQ level organization (rather than a 'sequencing library' level).
+The common scenario of sequencing the same 'sequencing library' more than once (in the above example) is the reason why the 10x technicians are moving towards a FASTQ-level organization (rather than a 'sequencing library' level).
 
 # Proposed 10X Technician Spreadsheets: FASTQ, Sample, Library Features, Features
-These are the proposed spreadsheets that will be used by the 10x techs (not all experiment-related columns are shown). Laura is working on making a new FASTQ-level spreadsheet - we are not considering the granular run-level and read-level FASTQs as separate FASTQs and in this document referring to this set as a "FASTQ" or "set of FASTQs". Below is the in-progress outline for this spreadsheet along with some example configurations:
+Below are proposed spreadsheets for use by the 10X techs (not all experiment-related columns are shown).
+
+## FASTQ-Level Spreadsheet
+Laura is working on making a new "FASTQ-oriented" spreadsheet, but that nomenclature is tricky: we're not considering individual run- and read-level FASTQs, but are referring to such a group collectively as a "FASTQ" or "set of FASTQs".
```

Comment:
my attempt at explaining this important but hard-to-name concept

Reply:
What do you mean by "run" level FASTQs? We are considering each set of FASTQs that come from the pooled libraries (e.g. the set of FASTQs that are demultiplexed from a BCL file). We are not explicitly considering the more granular FASTQs that distinguish sequencer-lane (*not 10x chip lane*) and read level information (e.g. R1,/R1 R2/I1, R3/R2 which are the actual segments of the molecule that are read and contain information on: cell/umi barcode, pooling index, and actual gene sequence) - as far as I understand. @Adeebius please feel free to correct anything.

Reply:
The phrase "run-level and read-level FASTQs" was here previously, and I didn't mean to change what was being said about it.

My impression is that "read-level" refers to FASTQs typically coming in `_R1` and `_R2` pairs (courtesy of paired-end sequencing).

I interpreted "run-level" to mean either "lane-level", like this part of the "2-sample" docs example:

![a group of 6 fastqs: 3 from each of 2 lanes](https://user-images.githubusercontent.com/465045/64629248-dc04e280-d3c0-11e9-9fc6-7e7ae0cbb7b4.png)

or "`Processing-Run`-level", like this part of the "1-sample" example:

![6 fastqs: 2 groups of 3 from different sequencing runs](https://user-images.githubusercontent.com/465045/64629371-13738f00-d3c1-11e9-9842-aaa7445dd77e.png)

I think I generally share your questions here and below about the different "set of FASTQs" concepts, and the need for better names for them. Hopefully we're close to converging on something.

Reply:
@ryan-williams ok, we can sort this out with @Adeebius